### PR TITLE
HELM-281: Create About Helm "welcome" dashboard

### PR DESCRIPTION
**Pull request by mmahacek**
# Pull Request Check Sheet

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new feature, is there documentation?
* [ ] If this is a new feature or substantial change, are there tests that cover it?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/HELM-281
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/opennms-helm)

### CODE REVIEW

**Comment by mmahacek** [comment]
@RangerRick There is a ` https://docs.opennms.com/helm/{helm-version}` I've included to use as a reference to the current Documentation URL. Is there a different/preferred way to indicate a placeholder to update that value on release?

File: src/dashboards/about.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        },
+        "gridPos": {
+          "h": 11,
+          "w": 18,
+          "x": 0,
+          "y": 0
+        },
+        "id": 2,
+        "options": {
+          "content": "# About Helm\n\nHELM is an application that allows users to create flexible dashboards to interact with data stored by OpenNMS. HELM works with Grafana to display and customize data from OpenNMS. Create and customize dashboards to display the data you want to see - alarms, outages, key performance indicators - in a way that best  meets the needs of your workflow and staff.\n\n## Datasources\n\n * Performance - retrieve performance metrics (i.e., data collection and/or response time metrics) from the Measurements API in OpenNMS.\n * Flow - retrieve application- and conversation-related metrics derived from flow data stored by OpenNMS.\n * Entity - retrieve entity model objects from OpenNMS. It currently supports alarms and nodes.\n\n## Example Dashboards\n\n * https://grafana.com/grafana/dashboards?search=opennms\n * https://github.com/opennms-forge/grafana-dashboards\n ",
```
Comment (RangerRick) [inline src/dashboards/about.json:35]: A nitpick, but there's really nowhere else (other than Jira issue names) that we refer to helm in all-caps.  I'd change it to always use "Helm"

Comment (mmahacek) [reply src/dashboards/about.json:35]: I actually copied that verbatim from the documentation.

Comment (RangerRick) [reply src/dashboards/about.json:35]: Oh weird. I guess I am wrong then. Guess we need to figure out what the branding is suppose to be then 'cause I've never referred to it that way publicly in anything. 😅

**Comment by RangerRick** [comment]
> @RangerRick There is a ` https://docs.opennms.com/helm/{helm-version}` I've included to use as a reference to the current Documentation URL. Is there a different/preferred way to indicate a placeholder to update that value on release?

That's a good question. Will probably need some kind of code to translate that during the build.  I would just hardcode it to `7.1.1-SNAPSHOT` for now and we can open an issue to make it dynamic.

**Review by mfuhrmann — APPROVED** [review]
The board works for me using Grafana 7.2.1 and HELM 6.
Really nice!

**Comment by mmahacek** [comment]
There are three main sections currently:

- Description of Helm and what data is provided by the included data sources.
- Links to documentation, Mattermost, and Discourse.
- Dynamic dashboard list that will show all dashboards in the current Grafana instance that have "OpenNMS" in their name.

Screenshot of what this looks like:
image

**Pull request merged**